<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Skill</class>
 <widget class="QDialog" name="Skill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>72</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/see);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>洞察术</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>可窥视对方手牌（冒险模式可用）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="see">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lv.5 可习得</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>72</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/double);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下回合对拼造成两倍HP伤害（双方有效）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>双龙出海</string>
   </property>
  </widget>
  <widget class="QPushButton" name="double_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lv.10 可习得</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>72</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/skill);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>技能学习</string>
   </property>
  </widget>
  <widget class="QLabel" name="tips">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>要学习该技能 请使用钢琴弹奏下面旋律：</string>
   </property>
  </widget>
  <widget class="QLabel" name="songNote">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>411</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/piano/mission);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tips_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(可以猜猜这段旋律是什么 不过没有奖励)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
